Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -95,6 +90,12 @@ class PlaceCarouselViewController: UIViewController {
         }
     }
 
+    // fake the location to Hilton Waikaloa Village, Kona, Hawaii
+    fileprivate var fakeLocation: CLLocation = CLLocation(latitude: 19.924043, longitude: -155.887652)
+
+    fileprivate var monitoredRegions: [String: GeofenceRegion] = [String: GeofenceRegion]()
+    fileprivate var timeAtLocationTimer: Timer?
```

Comment:
So, these should no longer be in `PlaceCarouselViewController` as all location monitoring has moved to `LocationMonitor` I'm guessing a rebase went awry